suite/src/__tests__/fast/model-correctness.test.ts

```diff
@@ -40,15 +39,21 @@ describe('Model Integration Test', () => {
   })
 
   test('Create a ModelInstanceDocument on one node and read it from another', async () => {
+    const isIndexed = await TestUtils.waitForConditionOrTimeout(async () => {
+      const indexedModels1 = await ceramicNode1.admin.getIndexedModels();
+      const indexedModels2 = await ceramicNode2.admin.getIndexedModels();
+      return indexedModels1.includes(modelId) && indexedModels2.includes(modelId);
+    }, 1000 * 60 * nodeSyncWaitTimeMin); // 1 minute timeout in millseconds
+    if (!isIndexed) {
+      throw new Error('Timeout reached: Failed to index the model')
+    }
     const modelInstanceDocumentMetadata = { model: modelId }
     const document1 = await ModelInstanceDocument.create(
       ceramicNode1,
       basicModelDocumentContent,
       modelInstanceDocumentMetadata,
     )
-    // We have to wait for some time for sync to happen
-    await delay(nodeSyncWaitTimeSec)
-    const document2 = await ModelInstanceDocument.load(ceramicNode2, document1.id)
+    const document2 = await loadDocumentOrTimeout(ceramicNode2, document1.id, 1000 * 60 * nodeSyncWaitTimeMin)
```

Comment:
2 minutes seems super long for a single event to sync.  Generally a document should sync in less than a second I would think.  Can we turn this down to like... 5 seconds?  Maybe 10 if you want to be conservative?

Reply:
yeah 2 minutes is a lot, i'll reduce it to 5 seconds. we were experiencing errors on a delay of 2 seconds. But that could've been cause the models were not indexed